query-engine/query-engine-node-api/src/engine.rs

```diff
@@ -319,12 +319,7 @@ impl QueryEngine {
 
             let query = RequestBody::try_from_str(&body, engine.engine_protocol())?;
 
-            let span = if tx_id.is_none() {
-                tracing::info_span!("prisma:engine", user_facing = true)
-            } else {
-                Span::none()
-            };
-
+            let span = tracing::info_span!("prisma:engine", user_facing = true);
```

Comment:
Related to Miguel's comment: With the duplication of most of the query engine surface API logic, we have 2 more occurrences of this code.

See [query-engine/src/server/mod.rs](https://github.com/prisma/prisma-engines/blob/fce8abaff49572845d2800356cd4cec8fd3dac83/query-engine/query-engine/src/server/mod.rs#L119-L125)

See [query-engine/query-engine-wasm/src/wasm/engine.rs](https://github.com/prisma/prisma-engines/blob/ca42e86a7b1bc9a4d32728da72fc42feaa05f011/query-engine/query-engine-wasm/src/wasm/engine.rs#L192-L196)

Reply:
Soon we'll have one more engine that's 99% duplicated as well :chef-kiss:

Reply:
I applied the changes from this PR and https://github.com/prisma/prisma-engines/pull/4154 to the WASM engine in the last commit.